R/predict.R

```diff
@@ -0,0 +1,104 @@
+#' Predict hazard, cumulative hazard or survival probability
+#'
+#' @param object An object of class \code{pam_xgb}
+#' @param newdata A data set containing the same covariates as used for model
+#' fitting. If of class \code{data.frame}, the function will transform the
+#' data to the PED format using the same settings as for the data used in
+#' estimation.
+#' @param type The type of prediction desired. Either hazard (\code{type = "hazard"}),
+#' cumulative hazard (\code{type = "cumu_hazard"}) or survival probability
+#' (\code{type = "surv_prob"}).
+#' @param ... Currently not used.
+#' @importFrom stats predict
+#' @return A matrix of predictions containing one row per
+#' observation (row in newdata) and 1 column per  specified time in the
+#' \code{times} argument.
+#' @seealso pamm
+#' @export
+predict.pamm <- function(
+  object,
+  newdata,
```

Comment:
should this have a `se` arg as well (and `se.type  = c("bla", "delta", "sim")`)?

Reply:
will try to implement. Is there a technical term for "bla". I usually call it direct transformation, but maybe there's a term for that?

Reply:
> Is there a technical term for "bla"

..comedy gold right here :rofl:   
`direct` would seem to be the more professional sounding choice.